schema/adminschema/v1.json

```diff
@@ -44,7 +44,26 @@
                   "type": "boolean"
               }
           }
-      }
+      },
+      "ClientCreateRequest": {
+          "id": "ClientCreateRequest",
+          "type": "object",
+	      "description": "'client' field is a client registration request as defined by the OpenID Connect dynamic registration spec, and holds fields such as redirect URLs, prefered algorithms, etc. For brevity field names and types of that object have been omitted.",
+          "properties": {
```

Comment:
Okay. We need to document that these exist btw; even I didn't know this generated a client. Would be a pity if we did all this work and no one used them.